lib/rules/declaration-block-trailing-semicolon/index.js

```diff
@@ -42,6 +42,10 @@ function rule(expectation, _, context) {
 		});
 
 		root.walkDecls((decl) => {
+			if (decl.parent.type !== 'rule' && decl.parent.type !== 'atrule') {
```

Comment:
I think we should use:

```js
if (decl.parent.type === 'object') return;
```

As not to exclude template literals, e.g.:

```js
const C = styled.a`color: red;`;
```

Reply:
Done